spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +350,21 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public ResponseEntity<String> deleteMultipleItems(String groupId, List<String> citationKeys, Long citationVersion) throws ZoteroConnectionException {
+        String citations = String.join(",", citationKeys);
```

Comment:
zotero accepts only 50 items at a time, so you need to make sure to make multiple requests if there are more than 50 items.